pkg/occlient/occlient.go

```diff
@@ -289,15 +289,6 @@ func New(skipConnectionCheck bool) (*Client, error) {
 	}
 	client.Namespace = namespace
 
-	// if we're not skipping the connection check, check the connection :)
-	if !skipConnectionCheck {
```

Comment:
Do we need this skipConnectionCheck variable anymore? We removed the check from where ever we can and now if its there at any place that would be a necessity.

Reply:
So can we remove this variable al all together?

Reply:
Addressed. PTAL.